jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -389,6 +397,17 @@ public Builder withZipkinSharedRpcSpan() {
       return this;
     }
 
+    /**
+     * If the exception is logged and 'message', 'stack' or 'error.kind' is missing. Tracer will
```

Comment:
`message`, `error.kind` could be done automatically,  and `stack` could be optional for perf reasons.